openff/qcsubmit/_tests/test_submissions.py

```diff
@@ -336,17 +359,23 @@ def test_basic_submissions_single_pcm_spec(fulltest_client):
     check_added_specs(ds=ds, dataset=dataset)
 
     for spec_name, spec in dataset.qc_specifications.items():
-        query = ds.iterate_records(
-            specification_names=spec_name,
+        query = list(
+            ds.iterate_records(
+                specification_names=spec_name,
+            )
         )
-        assert len(list(query)) == 1  # only used 1 molecule above
+        assert len(query) == 1  # only used 1 molecule above
         for name, _, record in query:
             assert record.status == RecordStatusEnum.complete
             assert record.error is None
             assert record.return_result is not None
             # make sure the PCM result was captured
-            assert record.extras["qcvars"]["PCM POLARIZATION ENERGY"] < 0
-            assert record.specification == spec
+            assert record.properties[solvent_energy] < 0
+            # make sure the correct solvent was used
+            assert solvent_evidence in record.stdout
+            assert record.specification.dict(
+                include={"method", "basis", "program"}
+            ) == spec.dict(include={"method", "basis", "program"})
```

Comment:
(non-blocking) I see that this used to be a direct equivalence check. Are there fields in the specs that differ now?